<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecuperarPassw</class>
 <widget class="QDialog" name="RecuperarPassw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_mail">
       <property name="text">
        <string>mail</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Boton_rec">
         <property name="text">
          <string>Recuperar contrase√±a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Boton_cancelar">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
